eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageTypes.scala

```diff
@@ -278,6 +278,8 @@ case class ChannelReady(channelId: ByteVector32,
   val alias_opt: Option[Alias] = tlvStream.get[ShortChannelIdTlv].map(_.alias)
 }
 
+case class Stfu(channelId: ByteVector32, initiator: Byte) extends LightningMessage
```

Comment:
We should use a `Boolean` here instead of a byte: see https://github.com/ACINQ/eclair/commit/e085b02359c0b692f199415a24064018f4fa2b80

Reply:
Fixed in [ab9cf21](https://github.com/ACINQ/eclair/pull/2680/commits/ab9cf211bade60f52c78971d771b983fff925e5b)